sdk/monitor/monitor-opentelemetry-exporter/src/utils/spanUtils.ts

```diff
@@ -230,7 +228,16 @@ function createDependencyData(span: ReadableSpan): RemoteDependencyData {
   }
   // DB Dependency
   else if (dbSystem) {
-    if (isSqlDB(String(dbSystem))) {
+    // TODO: Remove special logic when Azure UX supports OpenTelemetry dbSystem
+    if (String(dbSystem) === DbSystemValues.MYSQL) {
```

Comment:
It looks like the value used by UX matches up with the values used by [dbSystem](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#notes-and-well-known-identifiers-for-dbsystem), so can we not just used `dbSystem` value unless if SQLdb?

Reply:
Well yes but if semantic convention values change this will stop working, that is why I added this code explicitly, once UX rely on semantic convention package we can remove this code